src/display/canvas.js

```diff
@@ -1349,19 +1365,60 @@ class CanvasGraphics {
       height = img.height;
     const fillColor = this.current.fillColor;
     const isPatternFill = this.current.patternFill;
-    const maskCanvas = this.cachedCanvases.getCanvas(
-      "maskCanvas",
-      width,
-      height
-    );
-    const maskCtx = maskCanvas.context;
-    putBinaryImageMask(maskCtx, img);
+    const objToCanvas = ctx.mozCurrentTransform;
```

Comment:
I know that you're just moving pre-existing code, but how about giving this variable a more appropriate name (since I don't see how `objToCanvas` actually makes sense)?